core/src/main/java/com/capitalone/dashboard/util/DashboardUtils.java

```diff
@@ -2,15 +2,38 @@
 
 import com.capitalone.dashboard.model.Collector;
```

Comment:
Why do we need the utils class?

Reply:
I had to move one method out into utils so it can be leveraged by the logic that links the collector items to the dashboard. This is to avoid code duplication.

Reply:
@tabladrum are you good here?